service-base/src/main/java/org/eclipse/hono/service/http/DefaultFailureHandler.java

```diff
@@ -39,6 +39,7 @@
  */
 public class DefaultFailureHandler implements Handler<RoutingContext> {
 
+    private static final String MSG_UNKNOWN_ERROR = "<unknown error>";
```

Comment:
still, IMHO this is not helpful at all and I would rather have an empty body in this case